integreat_compass/cms/utils/email_utils.py

```diff
@@ -81,3 +84,35 @@ def send_activation_mail(user):
         )
     except BadHeaderError as e:
         logger.exception(e)
+
+
+def send_reset_mail(user):
+    """
+    Send a passwordf reset email to a user
+
+    :param user: the user
+    :type user: ~integreat_compass.cms.models.user.User
+    """
+    context = {"user": user}
+    subject = _("Integreat Compass | Your requested password reset")
+
+    token = password_reset_token_generator.make_token(user)
+    uid = urlsafe_base64_encode(force_bytes(user.pk))
+
+    subject += _("Reset your password")
```

Comment:
AFAICT there is no space when concatenating the strings `"Integreat Compass | Your requested password reset"` and `"Reset your password"`. Do I miss something?

Reply:
Uh, yes. I'm actually not sure why this is there, so I removed it. This part was copied from Integreat, I think it might worth checking out if the same thing is happening over there?